rel/config.exs

```diff
@@ -60,7 +60,6 @@ environment :dev do
   # dev mode.
   set(dev_mode: true)
   set(include_erts: true)
-  set(cookie: :"bAFS)[qmAg0XFS5`7Ncbz,$J:K3Tv)t@{OIPWJb`;N!k%z^H,^(L}:qI_$Tp8>EX")
```

Comment:
is this change related?

Reply:
No. But it was breaking the CLI remote console functionality.